app/cast_utils.py

```diff
@@ -124,17 +124,24 @@ def create_hibernation_node(cluster_id: str, castai_api_token: str, instance_typ
         return nodeId
 
 
-def delete_all_pausable_nodes(cluster_id: str, castai_api_token: str, hibernation_node_id: str, job_node_id=None):
+def delete_all_pausable_nodes(cluster_id: str, castai_api_token: str, hibernation_node_id: str, protect_eviction_disabled: bool, job_node_id=None):
```

Comment:
I would recommend to use same words as in k8s label removal-disabled not eviction -> protect_removal_disabled

Reply:
Updated